compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -2157,9 +2163,22 @@ static LogicalResult setConvRootConfig(mlir::FunctionOpInterface entryPointFn,
     scalableTileFlags.emplace_back(numTilingDims, false);
   }
 
+  DictionaryAttr pipelineConfig;
+  if (vecPreProcStrategy == VectorPreProcStrategy::Peeling) {
+    // Enable peeling. To this end, attach extra info to the pipeline config.
+    // This will later be extracted by LLVMCPULowerExecutableTargetPass.
+    auto ctx = convOp.getContext();
+    SmallVector<NamedAttribute> attrs;
+    auto peelAttrName = StringAttr::get(ctx, "enable_peeling");
```

Comment:
Instead of having a magic string floating in different files, how about adding a `getPeelAttrName` to CPUUtils?